src/rebar_dir.erl

```diff
@@ -103,12 +103,20 @@ home_dir() ->
     {ok, [[Home]]} = init:get_argument(home),
     Home.
 
+
 %% @doc returns the directory where the global configuration files for rebar3
 %% may be stored.
 -spec global_config_dir(rebar_state:t()) -> file:filename_all().
 global_config_dir(State) ->
-    Home = home_dir(),
-    rebar_state:get(State, global_rebar_dir, filename:join([Home, ".config", "rebar3"])).
+    rebar_state:get(State, global_rebar_dir, filename:join([rebar_config_dir(), ".config", "rebar3"])).
+
+rebar_config_dir() ->
+    case os:getenv("REBAR_CONFIG_DIR") of
+        C when C == false ->
```

Comment:
`false -> ...`

Reply:
Derp. Yeah, still want to see if this worth doing first. That is to say, I'm pretty sure it is, but pretty sure != sure. I'd hate for this to be added in a reactive matter.

Reply:
It's likely a good thing to make env-specific or profile-based global configurations.